src/inference/src/dev/core_impl.cpp

```diff
@@ -595,19 +595,7 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model_with_preprocess(ov::Pl
                                                                           const std::shared_ptr<const ov::Model>& model,
                                                                           const ov::RemoteContext& context,
                                                                           const ov::AnyMap& config) const {
-    std::shared_ptr<const ov::Model> preprocessed_model = model;
-
-    if (!is_new_api() && !std::dynamic_pointer_cast<InferenceEngine::IPluginWrapper>(plugin.m_ptr)) {
-        ov::pass::Manager manager;
-        manager.register_pass<ov::pass::AddPreprocessing>();
-
-        auto cloned_model = model->clone();
-        manager.run_passes(cloned_model);
-        preprocessed_model = cloned_model;
-    }
-
-    return context._impl ? plugin.compile_model(preprocessed_model, context, config)
-                         : plugin.compile_model(preprocessed_model, config);
+    return context._impl ? plugin.compile_model(model, context, config) : plugin.compile_model(model, config);
 }
```

Comment:
ilya, where is PPP handled in the case of old api call + cpu 2.0 plugin?

Reply:
This PR move all plugins to use 2.0 pre-processing. No difference between old and new cases